content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -104,18 +105,20 @@ istio-pilot-58c65f74bc-2f5xn             1/1       Running   0          1m
 
 * For option 2, uninstall using Helm:
 
+> Uninstalling this chart does not delete Istio's registered CRDs. Istio, by design, expects
+> CRDs to leak into the Kubernetes environment. As CRDs contain all runtime configuration
+> data in custom resources the Istio designers consider better to explicitly delete this
+> configuration rather then unexpectedly to lose it.
+
     {{< text bash >}}
     $ helm delete --purge istio-minimal
+    $ helm delete --purge istio-init
     {{< /text >}}
 
-    If your Helm version is less than 2.10.0, then you need to manually cleanup extra job resource before redeploy new version of Istio chart:
-
-    {{< text bash >}}
-    $ kubectl -n istio-system delete job --all
-    {{< /text >}}
+* If desired, run the following command to delete all CRDs:
 
-* If desired, delete the CRDs:
+> {{< warning_icon >}} Deleting CRDs deletes any configuration changes that you have made to Istio.
 
     {{< text bash >}}
-    $ kubectl delete -f install/kubernetes/helm/istio/templates/crds.yaml
+    $ for i in install install/kubernetes/helm/istio-init/files/*crd-1*yaml; do kubectl delete -f $i; done
```

Comment:
again `crd-*` should be sufficient.  We definately dont want a  wildcard on the beginning of the file.  the 1 is unnecessary.

Reply:
Done.